composer.lock

```diff
@@ -2321,6 +2321,101 @@
                 "issues": "https://www.drupal.org/project/issues/ctools"
             }
         },
+        {
+            "name": "drupal/eu_cookie_compliance",
```

Comment:
These changes should not be added either as you removed the composer.json related changes.